[2024-02-28T14:12:05.482+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_snowflake_etl.transform_data manual__2024-02-28T14:04:06.754502+00:00 [queued]>
[2024-02-28T14:12:05.511+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_snowflake_etl.transform_data manual__2024-02-28T14:04:06.754502+00:00 [queued]>
[2024-02-28T14:12:05.513+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2024-02-28T14:12:05.549+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_data> on 2024-02-28 14:04:06.754502+00:00
[2024-02-28T14:12:05.583+0000] {standard_task_runner.py:60} INFO - Started process 63 to run task
[2024-02-28T14:12:05.613+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gcs_snowflake_etl', 'transform_data', 'manual__2024-02-28T14:04:06.754502+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/GCS_to_Snowflake_DAG.py', '--cfg-path', '/tmp/tmppfch0uld']
[2024-02-28T14:12:05.689+0000] {standard_task_runner.py:88} INFO - Job 198: Subtask transform_data
[2024-02-28T14:12:05.889+0000] {task_command.py:423} INFO - Running <TaskInstance: gcs_snowflake_etl.transform_data manual__2024-02-28T14:04:06.754502+00:00 [running]> on host 1753c686362f
[2024-02-28T14:12:06.135+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='gcs_snowflake_etl' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T14:04:06.754502+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-28T14:04:06.754502+00:00'
[2024-02-28T14:12:07.617+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-28T14:12:07.632+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gcs_snowflake_etl, task_id=transform_data, execution_date=20240228T140406, start_date=20240228T141205, end_date=20240228T141207
[2024-02-28T14:12:07.668+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-28T14:12:07.717+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
